civiclens/nlp/pipeline.py

```diff
@@ -0,0 +1,89 @@
+import datetime
+
+from ..utils.database_access import Database, pull_data
+from . import comments, titles
+
+
+def doc_generator(df):
+    for row in df.iter_rows():
+        yield (row)
+
+
+if __name__ == "__main__":
+    # get time nlp last updated
+    nlp_updated_query = """SELECT nlp_last_updated
+                        FROM regulations_nlpoutput
+                        ORDER BY nlp_last_updated ASC LIMIT 1"""
+
+    db_date = Database()
+    last_updated = pull_data(
+        query=nlp_updated_query,
+        connection=db_date.conn,
+        schema=["nlp_last_updated"],
+    )[0, "nlp_last_updated"]
+
+    if last_updated is not None:
+        last_updated = last_updated.strftime("%Y-%m-%d %H:%M:%S")
+
+    # what docs dont need new titles
+    titles_query = """SELECT document_id
```

Comment:
also picky, but it'd be a little cleaner to wrap these as functions like `get_new_titles()` opposed to having stray SQL statements floating around